src/main/java/org/datadog/jmxfetch/Configuration.java

```diff
@@ -45,6 +50,44 @@ public String toString() {
     private Boolean hasInclude() {
         return getInclude() != null && !getInclude().isEmptyFilter();
     }
+    
+    /** Resolve dynamic tags for this configuration using cached values. */
```

Comment:
Could you please make the comment more explicit? What does "Resolve" mean in this case, what are the inputs and outputs?